src/PrimaryAuthController.js

```diff
@@ -97,6 +97,12 @@ function (Okta, PrimaryAuthForm, CustomButtons, FooterRegistration, PrimaryAuthM
       },
       'focusout input': function (e) {
         $(e.target.parentElement).removeClass('focused-input');
+      },
+      'click .button-show': function () {
+        var track = this.settings.get('tracker.track');
+        if(track) {
```

Comment:
Nit: maybe it would be nice to also check that `track` is function? With `_.isFunction()` for example